babeld/src/ubus.c

```diff
@@ -80,9 +82,19 @@ static int babeld_ubus_add_interface(struct ubus_context *ctx_local,
   if (!tb[INTERFACE_IFNAME])
     return UBUS_STATUS_INVALID_ARGUMENT;
 
+  if (tb[INTERFACE_RXCOST]) {
+    int cost = blobmsg_get_u32(tb[INTERFACE_RXCOST]);
+    if_conf = calloc(1, sizeof(struct interface_conf));
+    if (if_conf == NULL)
+      return UBUS_STATUS_UNKNOWN_ERROR;
+    if_conf->cost = cost;
+    //merge_ifconf(if_conf, if_conf, default_interface_conf);
+    //add_ifconf(if_conf, &interface_confs);
```

Comment:
I thought this is necessary, however if I do it babeld crashes. Without the functions, everything works perfectly. @jech could you help?